crates/api_common/src/community.rs

```diff
@@ -46,12 +41,6 @@ pub struct CommunityResponse {
   pub community_view: CommunityView,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
-/// An unfollow will return None
-pub struct FollowCommunityResponse {
```

Comment:
This isn't necessary... I found that when developing for the UI, the community sidebar takes in `CommunityView`, so its better to have the Pending state included in that anyway.